bigquery_etl/metadata/validate_metadata.py

```diff
@@ -27,6 +32,54 @@ def validate_public_data(metadata, path):
     return is_valid
 
 
+def validate_change_control(
+    file_path,
+    metadata,
+    codeowners_file,
+    project_id="moz-fx-data-shared-prod",
+    sql_dir="sql",
+):
+    """Verify that a query is correctly setup for change control."""
+    path_to_add = file_path.partition(f"{project_id}/")[2]
+    path_in_codeowners = os.path.join(sql_dir, project_id, path_to_add)
+    has_codeowner = CHANGE_CONTROL_LABEL in metadata.labels
+
+    if has_codeowner:
+        # This label requires to have at least one owner in the metadata file.
+        # And for any of the owners, at least one entry in the CODEOWNERS file.
```

Comment:
Yes, in this case, the team includes people from both teams and that is aligned with the requirement.
Thank you for the link, noted in case the requirement changes in the future.